server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -1205,6 +1211,23 @@ public static void toXContent(IndexMetaData indexMetaData, XContentBuilder build
             }
             builder.endArray();
 
+/*
+            ==== for the above, ClusterState.toXContent has the following ====
```

Comment:
sure, but how is this rendered differently? Can we see an example?

Reply:
Yes it's available here as an intermediate commit https://github.com/elastic/elasticsearch/pull/48924/commits/4850accccfb0a9d0c8cc810cb6813ec07e1e8e47 . Please let me know if this looks good.